symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -3,17 +3,21 @@ bdk:
   host: acme.symphony.com
   bot:
     username: mybot
-    privateKeyPath: ${user.home}/.symphony/keys/rsa-private-bot.pem
+    privateKeyPath: ${user.home}/.symphony/privatekey.pem
 
   app:
     appId: myapp
-    privateKeyPath: ${user.home}/.symphony/keys/app_privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/app_privatekey.pem
```

Comment:
You can actually reuse the same `privatekey.pem` for app auth as well, juste upload the same `publickey.pem` in ACP